### DISCUSSION TITLE
Welcome to SQL-Server-Fill-Factor-Determination Discussions!

### DISCUSSION THREAD

**Discussion by MByrdTX** [MByrdTX/SQL-Server-Fill-Factor-Determination#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MByrdTX** [comment]
It's been about 20 months since I've started this project.  It took me about 15 months to get all the details worked out for the code that is now in existence.   I've had no failures since August 2020 and review the email report each night.  Overall performance of the DB is exactly where I want it no apparent effect of DB performance.   I would love to see what y'all are seeing in your applications.  I'm always ready to answer questions as well as provide some technical assistance when needed.  Hope y'all have a Happy Holiday season and my prayers are for a much better 2021 for all.
Cheers,
Mike

**Comment by jefflomax** [comment]
There is a Pull Request that you created in 2019 -- not merged #1

**Comment by diablojeep** [comment]
I updated the code and have it working on my production systems.  It is doing a great job.  I'd like to limit it a bit and make it more like Ola Hallengren's scripts  But it is doing the work of compressing, tweaking the fill factor on indexes and rebuilding the top 20 (top 10 or top 1) in some cases as on my server it takes very long to do the work depending on how many indexes are selected.
